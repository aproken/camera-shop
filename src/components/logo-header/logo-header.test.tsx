import { render, RenderResult } from '@testing-library/react';
import LogoHeader from './logo-header';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Component: Footer', () => {
  it('Тестирование компонента Footer', () => {
    const { container }: RenderResult = render(
      <Router>
        <LogoHeader />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
