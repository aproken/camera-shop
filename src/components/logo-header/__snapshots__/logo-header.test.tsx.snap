// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer Тестирование компонента Footer 1`] = `
<div>
  <a
    aria-label="Переход на главную"
    class="header__logo"
    href="/"
  >
    <svg
      aria-hidden="true"
      height="36"
      width="100"
    >
      <use
        xlink:href="#icon-logo"
      />
    </svg>
  </a>
</div>
`;
